### PR TITLE
Update error message

### PR DESCRIPTION
**Pull request by TabithaKavyu**
Project Form: Update error message to be more clear #110

### CODE REVIEW

File: src/pages/AddProject.tsx [hunk 1]
```diff
@@ -231,8 +231,13 @@ class AddProject extends React.Component<RouteComponentProps, FormState> {
         </div>
 
         {(this.state.serverError.message) && <div className="error-message">
-          <p>{this.state.serverError.message}</p>
```
Comment (nurmerey) [inline src/pages/AddProject.tsx:234]: Let's make sure to keep a server-side error message on L234!

**Review by nurmerey — CHANGES_REQUESTED** [review]
Looking good so far! Left one little tiny ask.

File: src/pages/AddProject.tsx (hunk 1 above)
Comment (TabithaKavyu) [reply src/pages/AddProject.tsx:234]: okay

**Review by nurmerey — APPROVED** [review]
💯  LGTM!

**Pull request merged**